SagaFlow.UI/src/state/SagaFlowState.ts

```diff
@@ -180,7 +233,17 @@ class SagaFlow
             hasErrorFetchingConfig: false,
             fetchConfigError: undefined,
             config
-        }))
+        }));
+        
+        setTimeout(() => {
```

Comment:
using svelte's `tick` Lifecyle event might be better here.

Reply:
Updated to use tick